assets/src/edit-story/panels/size.js

```diff
@@ -28,51 +29,53 @@ function SizePanel( { selectedElements, onSetProperties } ) {
 	};
 	return (
 		<Panel onSubmit={ handleSubmit }>
-			<Title>
+			<PanelTitle>
 				{ 'Size' }
-			</Title>
-			<InputGroup
-				label="Width"
-				value={ state.width }
-				isMultiple={ width === '' }
-				onChange={ ( value ) => {
-					const ratio = width / height;
-					const newWidth = isNaN( value ) || value === '' ? '' : parseFloat( value );
-					setState( {
-						...state,
-						width: newWidth,
-						height: typeof newWidth === 'number' && lockRatio ? newWidth / ratio : height,
-					} );
-				} }
-				postfix="px"
-				disabled={ isFullbleed }
-			/>
-			<InputGroup
-				label="Height"
-				value={ state.height }
-				isMultiple={ height === '' }
-				onChange={ ( value ) => {
-					const ratio = width / height;
-					const newHeight = isNaN( value ) || value === '' ? '' : parseFloat( value );
-					setState( {
-						...state,
-						height: newHeight,
-						width: typeof newHeight === 'number' && lockRatio ? newHeight * ratio : width,
-					} );
-				} }
-				postfix="px"
-				disabled={ isFullbleed }
-			/>
-			<InputGroup
-				type="checkbox"
-				label="Keep ratio"
-				value={ lockRatio }
-				isMultiple={ false }
-				onChange={ ( value ) => {
-					setLockRatio( value );
-				} }
-				disabled={ isFullbleed }
-			/>
+			</PanelTitle>
+			<PanelContent>
+				<InputGroup
+					label="Width"
+					value={ state.width }
+					isMultiple={ width === '' }
+					onChange={ ( value ) => {
+						const ratio = width / height;
+						const newWidth = isNaN( value ) || value === '' ? '' : parseFloat( value );
+						setState( {
+							...state,
+							width: newWidth,
+							height: typeof newWidth === 'number' && lockRatio ? newWidth / ratio : height,
+						} );
+					} }
+					postfix="px"
+					disabled={ isFullbleed }
+				/>
+				<InputGroup
+					label="Height"
+					value={ state.height }
+					isMultiple={ height === '' }
+					onChange={ ( value ) => {
+						const ratio = width / height;
+						const newHeight = isNaN( value ) || value === '' ? '' : parseFloat( value );
+						setState( {
+							...state,
+							height: newHeight,
+							width: typeof newHeight === 'number' && lockRatio ? newHeight * ratio : width,
+						} );
+					} }
+					postfix="px"
+					disabled={ isFullbleed }
+				/>
+				<InputGroup
+					type="checkbox"
+					label="Keep ratio"
+					value={ lockRatio }
+					isMultiple={ false }
```

Comment:
Why can't we support multiple here?

Reply:
I didn't write this, just indented the section. Out of scope for this PR.